config/mainCfg_MuMu_UL18.cfg

```diff
@@ -7,7 +7,7 @@ data = DsingleMuA, DsingleMuB, DsingleMuC, DsingleMuD
 
 signals = GGF_Radion250, GGF_Radion260, GGF_Radion270, GGF_Radion280, GGF_Radion300, GGF_Radion320, GGF_Radion350, GGF_Radion400, GGF_Radion450, GGF_Radion500, GGF_Radion550, GGF_Radion600, GGF_Radion650, GGF_Radion700, GGF_Radion750, GGF_Radion800, GGF_Radion850, GGF_Radion900, GGF_Radion1000, GGF_Radion1250, GGF_Radion1500, GGF_Radion1750, GGF_Radion2000, GGF_Radion2500, GGF_Radion3000
```

Comment:
Technically there is no need to process all the signals for this channel, unless we want to use it in the fit.